pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -74,16 +77,104 @@ type cloudWatchExecutor struct {
 
 func (e *cloudWatchExecutor) newSession(region string) (*session.Session, error) {
 	dsInfo := e.getDSInfo(region)
-	creds, err := getCredentials(dsInfo)
+
+	bldr := strings.Builder{}
+	for i, s := range []string{dsInfo.AuthType.String(), dsInfo.AccessKey, dsInfo.Profile, dsInfo.AssumeRoleARN} {
```

Comment:
Region is missing. See also [this](https://github.com/grafana/grafana/pull/25548#discussion_r500076256) discussion.

Reply:
@papagian Thanks, I forgot to add it apparently.